tensorboard/data/server/data_compat.rs

```diff
@@ -38,6 +38,12 @@ pub(crate) mod plugin_names {
     pub const GRAPH_RUN_METADATA: &str = "graph_run_metadata";
     pub const GRAPH_RUN_METADATA_WITH_GRAPH: &str = "graph_run_metadata_graph";
     pub const GRAPH_KERAS_MODEL: &str = "graph_keras_model";
+    pub const HISTOGRAMS: &str = "histograms";
+    pub const TEXT: &str = "text";
+    pub const PR_CURVES: &str = "pr_curves";
+    pub const HPARAMS: &str = "hparams";
+    pub const MESH: &str = "mesh";
+    pub const NPMI: &str = "npmi";
```

Comment:
I think [nPMI data has always had `DATA_CLASS_TENSOR`][1] at rest. Can
we get away with dropping the special case here?

[1]: https://github.com/tensorflow/tensorboard/blame/1e7a722c49829351235308d8fdc31e28d4a04383/tensorboard/plugins/npmi/metadata.py#L45